.devcontainer/Dockerfile-actual

```diff
@@ -9,13 +9,19 @@ COPY llvm.patch /llvm-build/
 RUN sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list \
     && apt update \
     && apt install -y --no-install-recommends \
-        ca-certificates gnupg build-essential dpkg-dev devscripts locales cmake make python3 zlib1g wget subversion \
-        unzip git \
+        ca-certificates gnupg build-essential dpkg-dev devscripts locales \
+        make python3 zlib1g wget subversion unzip git curl \
```

Comment:
subversion is really needed? for what?

Reply:
Probably copied from an old build instruction from somewhere. I don't feel it's worth testing as it takes 40+ minutes just to test it 🤣 Anyway it's only there in the first stage when we're compiling the toolchains, and the second stage (`# 2nd image: devcontainer`) resets everything and there's no svn in the final image.